etc/p3-config/p3-config.yml

```diff
@@ -420,13 +421,131 @@ p3_bdn_router:
   interfaces:
     - "{{ p3_subnet_bdn.name }}"
 
+# Octavia network name.
+octavia_network_name: "octavia"
+
+# Octavia network.
+octavia_network:
+  name: "{{ octavia_network_name }}"
+  project: "service"
+  provider_network_type: "vlan"
+  provider_physical_network: "physnet1"
+  provider_segmentation_id: 148
+  shared: False
+  # Subnet configuration.
+  subnets:
+    - "{{ octavia_subnet }}"
+
+# Octavia subnet.
+octavia_subnet:
+  name: "{{ octavia_network_name }}"
+  project: "service"
+  cidr: "10.48.0.0/16"
+  enable_dhcp: true
+  allocation_pool_start: "10.48.253.2"
+  allocation_pool_end: "10.48.253.254"
+
+# P3 security groups.
+p3_security_groups:
+  - "{{ octavia_sg }}"
+
+# Octavia security group.
+octavia_sg:
+  name: octavia-sg
+  description: "Octavia security group"
+  project: service
+  rules:
+    - protocol: udp
+      port_range_min: 5555
+      port_range_max: 5555
+      remote_ip_prefix: "0.0.0.0/0"
+    - protocol: udp
+      port_range_min: 10514
+      port_range_max: 10514
+      remote_ip_prefix: "0.0.0.0/0"
+    - protocol: udp
+      port_range_min: 20514
+      port_range_max: 20514
+      remote_ip_prefix: "0.0.0.0/0"
+    - protocol: tcp
+      port_range_min: 9443
+      port_range_max: 9443
+      remote_ip_prefix: "0.0.0.0/0"
+    - protocol: tcp
+      port_range_min: 22
+      port_range_max: 22
+      remote_ip_prefix: "10.208.0.0/30"
+    - protocol: icmp
+      remote_ip_prefix: "0.0.0.0/0"
+
 ###############################################################################
 # Configuration of nova flavors for P-cubed.
 
 # List of nova flavors in the p3 project. Format is as required by the
 # stackhpc.os-flavors role.
 p3_flavors:
   - "{{ p3_flavor_compute_A }}"
+  - "{{ p3_flavor_xlarge }}"
+  - "{{ p3_flavor_large }}"
+  - "{{ p3_flavor_medium }}"
+  - "{{ p3_flavor_small }}"
+  - "{{ p3_flavor_tiny }}"
+  - "{{ p3_flavor_octavia }}"
+
+p3_flavor_xlarge:
+  name: "v1.xlarge"
+  vcpus: 16
+  ram: 16384
+  disk: 320
```

Comment:
looks like tehre are 24 cores and 252G of RAM.

Reply:
I have copied over the same config as IRIS.

Reply:
so the host is a difference size, so the IRIS flavor are not going to work. I guess its the previous generation with two rather than three memory controllers per socket.